packages/backend/discovery/arbitrum/ethereum/diffHistory.md

```diff
@@ -1,3 +1,69 @@
+# Diff at Mon, 13 Nov 2023 15:23:11 GMT:
+
+- author: Adrian Adamiak (<adrian@adamiak.net>)
+- comparing to: master@a45348c9ae2e765b872be3f217168f73b62d35a6
+
+## Description
+
+Transactions were executed via L1ArbitrumTimelock to replace some of the Security Council members.
+
+Names of the owners of the EOAs listed below were manually found on public Arbitrum forums,
+so they may be incorrect.
+
+Removed EOAs:
+
+- 0x526C0DA9970E7331d171f86AeD28FAFB5D8A49EF (Mo Dong?)
+- 0x0E5011001cF9c89b0259BC3B050785067495eBf5 (Diane Dai?)
+- 0x8688515028955734350067695939423222009623 (Celeb Lau?)
+- 0x88910996671162953E89DdcE5C8137f9077da217 (??)
+- 0x8e6247239CBeB3Eaf9d9a691D01A67e2A9Fea3C5 (Bryan Pellegrino?)
+
+Added EOAs:
+
+- 0x8F10e3413586c4a8DCfcE19D009872b19e9cd8E3 (Patrick McCorry?)
```

Comment:
confirming that these IDs are all correct (can check it against the tally UI [here](https://www.tally.xyz/gov/arbitrum/council/security-council/election/0/round-2))